tests/helpers.py

```diff
@@ -2,41 +2,11 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import datetime
 from uuid import uuid4
 
 from celery.result import AsyncResult
 
 
 class SimpleFakeTaskBase(object):
-    def apply_async(self, args=None, kwargs=None, task_id=None, producer=None, link=None, link_error=None, **options):
+    def apply_async(self, *args, **kwargs):
```

Comment:
We don't really care about the specific arguments, so this just adds verbosity (but accepts all the same arguments as the original version).

Also, this won't bite us in future if the original `Task`'s signature ever changes.